Minor attempt to clear up the behaviour in the documentation (#372)
xataio/pgroll: docs/README.md

$ pgroll start sql/03_add_column.json
```

This starts the migration defined in the `sql/03_add_column.json` file.

> After starting a migration there will be two schema versions in the database; one for the old schema before the migration (e.g. `public_02_create_table`) and one for the new version with the schema changes (e.g. `public_03_add_column`). At this point, the earlier version will be still available under the main schema name (e.g. `public`).

Reviewer: I think it's important to make it clear that the versioned schema (like `public_02_create_table`) contain views on the tables in the `public` schema. Applications using schema managed by`pgroll` shouldn't connect directly to `public` but rather use the versioned views.

```suggestion
After starting a migration there will be two schema versions in the database; one for the old schema before the migration (e.g. `public_02_create_table`) and one for the new version with the schema changes (e.g. `public_03_add_column`).  Each schema contains views on the tables in the `public` schema.
```

Running `pgroll complete` when there is no migration in progress is a no-op.

> Completing a `pgroll` migration removes the previous schema version from the database (e.g. `public_02_create_table`), leaving only the latest version of the schema (e.g. `public_03_add_column`), available under both specific and main schema name (i.e. both `public_03_add_column` and `public`).

Reviewer: Another attempt to make it clear that the version schema contain views and the base schema (default `public`) contains the tables.

```suggestion
Completing a `pgroll` migration removes the previous schema version from the database (e.g. `public_02_create_table`), leaving only the latest version of the schema (e.g. `public_03_add_column`). At this point, any temporary columns and triggers created on the affected tables in the `public` schema will also be cleaned up, leaving the table schema in its final state.
```
